Lib/ufo2ft/featureWriters/markFeatureWriter.py

```diff
@@ -539,17 +626,22 @@ def _makeMarkToMarkLookup(
         return lkp
 
     def _makeMarkFeature(self, include):
-        baseLkp = self._makeMarkLookup(
-            "mark2base", self.context.markToBaseAttachments, include
-        )
+        baseLkps = []
+        for i, attachments in enumerate(self.context.groupedMarkToBaseAttachments):
+            lookup = self._makeMarkLookup(
+                f"mark2base{'_' + str(i) if i > 0 else ''}", attachments, include
+            )
+            if lookup:
+                baseLkps.append(lookup)
+        # TODO: do the same for ligaLkps?
         ligaLkp = self._makeMarkLookup(
             "mark2liga", self.context.markToLigaAttachments, include
         )
-        if baseLkp is None and ligaLkp is None:
+        if not baseLkps and ligaLkp is None:
             return
 
         feature = ast.FeatureBlock("mark")
-        if baseLkp:
+        for baseLkp in baseLkps:
             feature.statements.append(baseLkp)
         if ligaLkp:
```

Comment:
I guess we are not also supporting multiple mark anchors for mark-to-liga lookups. Is that intentional? Less frequent, maybe

Reply:
I left that one out because I had no idea whether it should apply there as well. I defer to maybe @khaledhosny to tell us if that's needed there too? If so, I'll apply to mark2liga the same changes I did to mark2base

Reply:
But I would be grateful for help with writing the test for that

Reply:
I don’t see why not, OpenType does not have any limitations here AFAIK.

Reply:
I added a test here: https://github.com/googlefonts/ufo2ft/pull/416/commits/a87a71abf337e36e0d47720b05e4b7ea691dc29f

Could you please have a look at it before I try to implement what I'm suggesting in the test?

Reply:
This is needed for mark2liga as much as for mark2base and mark2mark. Any glyph (base, mark or ligature) should be able to use different anchors instead of just one.